### PR TITLE
js2 week2 homework

### PR DESCRIPTION
**Pull request by KavyaReddy732**
Hope i write the code in a proper way.

### CODE REVIEW

File: javascript/javascript2/week2/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+let numbers = [1, 2, 3, 4,5,6,7];
+//let newNumbers = [];
+
+let newNumbers = numbers.filter(element => element % 2 !==0)
+                        .map(item => item * 2);
```
Comment (NayanaKamtekar) [inline javascript/javascript2/week2/index.js:5]: Perfect

File: javascript/javascript2/week2/movies.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...and bleu","year": 1988,"rating": 7.6,"votes": 41252,"running_times": 10080},{"title": "Le locataire","year": 1976,"rating": 7.8,"votes": 30919,"running_times": 7560},{"title": "Le mépris","year": 1963,"rating": 7.7,"votes": 22380,"running_times": 6180},{"title": "Le procès","year": 1962,"rating": 7.8,"votes": 15401,"running_times": 7140},{"title": "Le salaire de la peur","year": 1953,"rating": 8.2,"votes": 40345,"running_times": 9360},{"title": "Le violon rouge","year": 1998,"rating": 7.7,"votes": 28394,"running_times": 8400},{"title": "Lean on Me","year": 1989,"rating": 7.4,"votes": 15573,"running_times": 6480},{"title": "Leap Year","year": 2010,"rating": 6.5,"votes": 78668,"running_times": 6000},{"title": "Leap of Faith","year": 1992,"rating": 6.1,"votes": 10526,"running_times": 6480},{"title": "Leatherface: Texas Chainsaw Massacre III","year": 1990,"rating": 5.2,"votes": 11025,"running_times": 5100},{"title": "Leatherheads","year": 2008,"rating": 6,"votes": 28907,"running_times": 6840},{"title": "Leaves of Grass","year": 2009,"rating": 6.5,"votes": 24966,"running_times": 6300},{"title": "Leaving Las Vegas","year": 1995,"rating": 7.6,"votes": 100091,"running_times": 6660},{"title
```
Comment (NayanaKamtekar) [inline javascript/javascript2/week2/movies.js:62]: Good work. but In No.7 Question expecting array of movies where a word in the title is duplicated and your solution gives a array of movies title which are duplicated.

**Review by NayanaKamtekar — COMMENTED** [review]
Good work.

File: javascript/javascript2/week2/hyf-bay/main.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+maxPriceInput.addEventListener('keyup', (e)=>{
+    const maxPrice = maxPriceInput.value;
+    const filterbyPrice = products.filter((element)=>{
+        return (element.price <= maxPrice);
+  });
+  productsUl.innerHTML = " ";
+  renderProducts(filterbyPrice);
+});
+
```
Comment (NayanaKamtekar) [inline javascript/javascript2/week2/hyf-bay/main.js:48]: Did you check those events in app????

Comment (KavyaReddy732) [reply javascript/javascript2/week2/hyf-bay/main.js:48]: yes, it's working

Comment (NayanaKamtekar) [reply javascript/javascript2/week2/hyf-bay/main.js:48]: Yes, it's working but still there is scope for improvement as it is not sorted as per sort  and ship to options.

**Pull request merged**